spec/till_spec.rb

```diff
@@ -0,0 +1,24 @@
+require 'till'
+
+describe Till do
+
+  let(:order         ) { double :order, items: { :burger => 9.99 } }
```

Comment:
Unnecessary spacing detected.<br>Use the new Ruby 1.9 hash syntax.<br>Space inside parentheses detected.